docs/source/operations/guide-troubleshooting.rst

```diff
@@ -25,6 +25,10 @@ When running Armada in the container you can execute docker logs to retrieve log
 
     docker logs [container-name | container-id]
 
+Errors/Exceptions
+-----------------
+
+A guide for interpreting errors/exceptions can be found `here <http://armada-helm.readthedocs.io/en/latest/operations/guide-api.html>`.
```

Comment:
you need a _ at the end of this to make a link.  

`here <http://armada-helm.readthedocs.io/en/latest/operations/guide-api.html>`_